api/preprints/views.py

```diff
@@ -75,22 +76,25 @@ def update(self, request, *args, **kwargs):
         preprint = self.get_preprint(check_object_permissions=False)
         if preprint.is_latest_version or preprint.machine_state == 'initial':
             return super().update(request, *args, **kwargs)
-
-        raise Conflict(detail='Edit availiabe only for the last preprint version or withdrawn preprints')
+        message = 'User can not edit previous versions of a preprint'
+        sentry.log_message(message)
+        raise Conflict(detail=message)
 
     def create(self, request, *args, **kwargs):
         preprint = self.get_preprint(check_object_permissions=False)
         if preprint.is_latest_version or preprint.machine_state == 'initial':
             return super().create(request, *args, **kwargs)
-
-        raise Conflict(detail='Edit availiabe only for the last preprint version or withdrawn preprints')
+        message ='User can not edit previous versions of a preprint'
```

Comment:
Fix `flake8`

> api/preprints/views.py:87:18: E225 missing whitespace around operator

For details, see https://github.com/CenterForOpenScience/osf.io/actions/runs/12411685125/job/34649900283?pr=10861#step:5:35